packages/web-components/src/components/content-group-horizontal/content-group-horizontal.ts

```diff
@@ -7,36 +7,28 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { css, customElement, html } from 'lit-element';
+import { customElement } from 'lit-element';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
-import DDSContentGroup from '../content-group/content-group';
+import deprecate from '@carbon/ibmdotcom-utilities/es/utilities/deprecate/deprecate';
+import DDSContentBlockHorizontal from '../content-block-horizontal/content-block-horizontal';
+
 import '../horizontal-rule/horizontal-rule';
-import styles from './content-group-horizontal.scss';
 
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
 /**
  * Horizontal version of content group.
  *
- * @element dds-content-group-horizontal
+ * @element dds-content-block-horizontal
  */
 @customElement(`${ddsPrefix}-content-group-horizontal`)
-class DDSContentGroupHorizontal extends DDSContentGroup {
-  render() {
-    return html`
-      ${super.render()}
-      <dds-hr></dds-hr>
-    `;
-  }
-
+class DDSContentGroupHorizontal extends DDSContentBlockHorizontal {
   static get stableSelector() {
     return `${ddsPrefix}--content-group-horizontal`;
   }
-
-  // `styles` here is a `CSSResult` generated by custom WebPack loader
-  static get styles() {
-    return css`${super.styles}${styles}`;
-  }
 }
-
-export default DDSContentGroupHorizontal;
+export default deprecate(
+  DDSContentGroupHorizontal,
+  'The ContentGroupHorizontal component has been deprecated in favor of the ContentBlockHorizontal component. ' +
+    'See ContentBlockHorizontal documentation for more information.'
```

Comment:
The component names should match our web components naming conventions here, i.e. `content-group-horizontal`.